Pipfile

```diff
@@ -33,6 +33,7 @@ wagtail-modeltranslation = "==0.10.1"
 cloudinary = "*"
 wagtail-experiments = "*"
 pyyaml = "==5.1"
+invoke = "*"
```

Comment:
This is not necessary, can you remove it? You will need to lock again.

Reply:
Actually, can you uncommit the pipfile and pipfile.lock from that PR? There's dependencies updates in it that I would like to handle myself. Thanks

Reply:
This got added when I fetched from upstream, let me remove it and then commit again

Reply:
Please, remove the lock file too.

Reply:
ok

Reply:
from your PR, not from the project...

Reply:
working on it

Reply:
made changes in Pipfile.lock as it was at initial

Reply:
is it alright now?